### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by sanjayss34** [BradyFU/Video-MME#27]
Hello, do you provide a validation/development set (separate from the test set)? This is so that we can evaluate our model as we are developing it, since tuning model design/hyperparameters according to the test set is undesirable.

**Comment by BradyFU** [comment]
Hi, thanks for your suggestion! There is no validation set yet. 
We hope that Video-MME can be directly used to evaluate the generalization ability.
